Sources/PointFree/Account/Account.swift

```diff
@@ -9,11 +9,11 @@ import Stripe
 import Tuple
 import Views
 
-let accountResponse: Middleware<StatusLineOpen, ResponseEnded, Tuple2<User?, SubscriberState>, Data> =
-  filterMap(require1 >>> pure, or: loginAndRedirect)
+let accountResponse//: M<Tuple2<User?, SubscriberState>>
```

Comment:
did you mean to drop the type and have this comment here?

Reply:
Most of the time, typing these middleware led to _slower_ builds (by 2x). I user comments to extract/simplify some of these. I'll delete this comment...